operation-domain/src/main/java/org/sopt/makers/operation/user/domain/SocialType.java

```diff
@@ -1,6 +1,15 @@
 package org.sopt.makers.operation.user.domain;
 
+import java.util.Arrays;
+
 public enum SocialType {
     GOOGLE,
-    APPLE,
+    APPLE;
+
+
+    public static boolean isContains(String type) {
+        SocialType[] socialTypes = SocialType.values();
+        return Arrays.stream(socialTypes)
+                .anyMatch(socialType -> socialType.name().equals(type));
```

Comment:
`P3`

지역 변수 선언을 한 이유가 따로 있는지 궁금합니다!!
물론 사소하지만 불필요한 변수 선언으로 저장 공간 낭비가 있을 것 같아서요!!

---
또한 String으로 `name()`과 `euals()` 연산하는 것보다
객체 비교를 하는 것이 논리적 오류 발생 위험성이 상대적으로 적다고 들었습니다!

아래와 같이 개선하면 어떨까 생각하는데 원용님의 의견이 궁금해요!!
```suggestion
        return Arrays.stream(SocialType.values())
                .anyMatch(socialType -> socialType.equals(SocialType.valueOf(type)));
```

Reply:
SocialType.valueOf 를 하지 않고 해당 메서드를 만든 이유는 type이 만약에 'GOOGLE', 'APPLE'이 아닌 다른 문자열이 들어왔을 때 IilgalException을 반환하기 때문입니다.

그래서 SocialType의 arrays를 받은 이후 해당 arrays 내에 값이 없으면 false를 반환하도록 구현했습니다

Reply:
좋아요!! 중간 연산에 대해서는 기존에 PR 올리신대로 처리해도 좋을 것 같습니다!!

++ 불필요한 메서드 내 지역변수 선언만 `stream()` 안에서 처리해주시면 감사드리겠습니다!! 🙇‍♂️